LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2018 Data-driven Computational Modelling
+Copyright (c) 2018-2021 petab package developpers
```

Comment:
I think in copyrights, the starting date is usually fine (in the sense of "holding the copy right since that date"), as otherwise the end date would of course always require updating (can be wrong there though).

Reply:
and devel-hoppers please.

Reply:
```suggestion
Copyright (c) 2018 petab package developers
```